sass/base/components/single-msg/_single-msg.scss

```diff
@@ -334,4 +335,31 @@
       }
     }
   }
+  &.single-msg--copy-only-large {
+    .single-msg__image {
+      display: none;
+    }
+    .single-msg__copy_wrapper {
+      background-color: transparent;
+      .single-msg__copy {
+        padding: 0 30px;
+
+        /* Remove margins from the last/only element in the body copy,
+          as to not mess up our vertical centering */
+        .single-msg__body :last-child {
+          margin-bottom: 0;
+        } 
+      }
+
+      @include breakpoint($screen-md) {
+        @include span(10);
+        @include push(1);
+        position: relative;
+        left: auto;
+        height: auto; //Does this need to match the height of the img version?
```

Comment:
did you get an answer to this question yet? I'd say keep the height auto to fit around the content. Or does design want a set height you think? Also, I think only the background image single message row has a set height, so I'd say keep it auto.

Reply:
I think it's fine to keep it `auto`. I have just added an extra variation. Didn't think about the question. Do you want me to remove the comment? height `auto` seems to be working fine with all variations.

Reply:
Well I think if it's not a question anymore we should remove it, not sure in how much of a hurry we are to get this in..